src/PipeLineTemplate/lib/notification.ts

```diff
@@ -0,0 +1,29 @@
+import * as cdk from "@aws-cdk/core";
+import * as chatbot from '@aws-cdk/aws-chatbot';
+import * as ssm from '@aws-cdk/aws-ssm';
+import { BuildOperationsDetails } from "./buildOperationsDetails";
+import { IPipeline } from "@aws-cdk/aws-codepipeline";
+
+export class Notification extends cdk.Construct {
+    constructor(scope: cdk.Construct, id: string, props: BuildOperationsDetails, pipeline: IPipeline) {
+        super(scope, id);
+        
+        let slackChannelName = 'sphinx-env-';
```

Comment:
can you read all constants from a parameter , which populates from ENV Variable at the entry point ?